Packages/org.plunderludics.UnityHawk/Runtime/Emulator/Emulator.cs

```diff
@@ -204,19 +204,25 @@ struct BizhawkArgs {
     }
     BizhawkArgs _currentBizhawkArgs; // remember the params corresponding to the currently running process
 
-    // Dictionary of registered methods that can be called from bizhawk lua
-    // bytes-to-bytes only rn but some automatic de/serialization for different types would be nice
+    /// TODO: bytes-to-bytes only rn but some automatic de/serialization for different types would be nice
```

Comment:
Not you but should probably be 'string-to-string' instead of 'bytes-to-bytes' here